<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-QTHT79U" ignored="0" name="Test" tests="15" failures="14" timestamp="18 May 2018 19:32:10 GMT" time="48.908" errors="0">
  <testcase name="testCase" time="8.993" classname="com.alg.test.executionengine.Executor"/>
  <testcase name="testCase" time="2.794" classname="com.alg.test.executionengine.Executor">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at com.alg.test.executionengine.Executor.testCase(Executor.java:169)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testCase -->
  <testcase name="testCase" time="2.727" classname="com.alg.test.executionengine.Executor">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at com.alg.test.executionengine.Executor.testCase(Executor.java:169)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testCase -->
  <testcase name="testCase" time="2.76" classname="com.alg.test.executionengine.Executor">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at com.alg.test.executionengine.Executor.testCase(Executor.java:169)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testCase -->
  <testcase name="testCase" time="2.768" classname="com.alg.test.executionengine.Executor">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at com.alg.test.executionengine.Executor.testCase(Executor.java:169)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testCase -->
  <testcase name="testCase" time="2.842" classname="com.alg.test.executionengine.Executor">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at com.alg.test.executionengine.Executor.testCase(Executor.java:169)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testCase -->
  <testcase name="testCase" time="2.753" classname="com.alg.test.executionengine.Executor">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at com.alg.test.executionengine.Executor.testCase(Executor.java:169)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testCase -->
  <testcase name="testCase" time="2.826" classname="com.alg.test.executionengine.Executor">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at com.alg.test.executionengine.Executor.testCase(Executor.java:169)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testCase -->
  <testcase name="testCase" time="2.726" classname="com.alg.test.executionengine.Executor">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at com.alg.test.executionengine.Executor.testCase(Executor.java:169)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testCase -->
  <testcase name="testCase" time="2.742" classname="com.alg.test.executionengine.Executor">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at com.alg.test.executionengine.Executor.testCase(Executor.java:169)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testCase -->
  <testcase name="testCase" time="2.578" classname="com.alg.test.executionengine.Executor">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at com.alg.test.executionengine.Executor.testCase(Executor.java:169)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testCase -->
  <testcase name="testCase" time="2.765" classname="com.alg.test.executionengine.Executor">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at com.alg.test.executionengine.Executor.testCase(Executor.java:169)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testCase -->
  <testcase name="testCase" time="2.724" classname="com.alg.test.executionengine.Executor">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at com.alg.test.executionengine.Executor.testCase(Executor.java:169)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testCase -->
  <testcase name="testCase" time="2.713" classname="com.alg.test.executionengine.Executor">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at com.alg.test.executionengine.Executor.testCase(Executor.java:169)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testCase -->
  <testcase name="testCase" time="2.751" classname="com.alg.test.executionengine.Executor">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at com.alg.test.executionengine.Executor.testCase(Executor.java:169)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testCase -->
</testsuite> <!-- Test -->
